docs/source/getting-started.mdx

```diff
@@ -46,6 +46,33 @@ To install a **specific version** of Rover (recommended for CI environments to e
 iwr 'https://rover.apollo.dev/win/v0.26.2' | iex
 ```
 
+#### Installing from a binary mirror using the bash and powershell scripts
+
+Both the bash and powershell scripts support the `APOLLO_ROVER_DOWNLOAD_GITHUB_HOST` environment variable,
+which defaults to `https://github.com/apollographql/rover/releases/download`.
+
+This can be set in your environment to specify a remote mirror or proxy to github release download
+url for the Apollo Rover GitHub repository.
```

Comment:
```suggestion
This can be set in your environment to specify a remote mirror or proxy to a GitHub release download URL for the Apollo Rover GitHub repository.
```